include/xtypes/Module.hpp

```diff
@@ -321,6 +372,7 @@ class Module
 protected:
     std::map<std::string, DynamicType::Ptr> constants_types_;
     std::map<std::string, DynamicData> constants_;
+    std::map<std::string, DynamicType::Ptr> enumerations_32_;
```

Comment:
You can store the Enum itself instead of the Ptr. Could be easier to use (for example, line 300)

Reply:
As talked, we will keep it as Ptr at the moment, but maybe we can change this in the future.